tutorial/README.md

```diff
@@ -0,0 +1,221 @@
+
+## What is NOT in this document?
+
+This tutorial will not teach you how to build a bundle in full. For details on how to write an Earthmover bundles with assessments in mind, see [Writing earthmover bundles to integrate assessment data into Ed-Fi](https://slite.com/api/public/notes/LKGIWG-7-2-rqD/redirect) .
+
+This tutorial will also not allow you to run lightbeam because there is no ODS that has corresponding sample data.
+
+## Prerequisites
+
+You should have working knowledge of the following:
+
+-   [YAML](https://realpython.com/lessons/yaml-introduction/)
+-   [Jinja](https://www.getdbt.com/dbt-learn/lessons/intro-to-jinja#1)
+-   JSON
+    -   jsont
+    -   jsonl
+
+# Step 0: Initial Setup
+
+-   Clone the repo <https://github.com/edanalytics/earthmover_edfi_bundles>
+-   `git checkout` a new branch for your tutorial work. There is no need to push this branch up to remote.
+-   `cd` to the `tutorial/` directory
+
+# Step 1: Introduction to the directory structure
+
+## Files in an EM bundle
+
+Each of these are files that you will need to populate for your EM/LB project. It's not a bad idea to copy and paste from existing bundles, both for inspiration and for consistency.
+
+-   `earthmover.yaml` - This is a config file that defines a list of "instructions" that are run on the data files before they get transformed by the template files.
+-   `lightbeam.yaml` - This is a lightbeam config file which is required to run lightbeam. Ignore for this tutorial, but it is worth noting that it exists in a bundle.
+-   `README.md` - The readme should include information on how to run this bundle, including instructions for configuration specific to the bundle.
+
+## Directories in an EM bundle
+
+-   Inputs
+    -   `data/` - This contains input data sources. Typically, these are your student assessment files. This directory may have a `.gitkeep` but it will typically be empty in remote. You don't want to push your student assessment files to remote. In the tutorial bundle, we've included a very simple fake assessment.
+    -   `seeds/` - This contains input data sources. You will be creating and populating these, and they should be applicable to any implementation of this bundle. Typically, these are assessment metadata (e.g., name, scores) and descriptors (e.g., performanceLevelDescriptors)
+-   Templates
+    -   `templates/` - This contains the jsont (json template) files that define how your flat files (in `data/` and `seeds/`) will get transformed into jsonl (json lines).
+-   Outputs
+    -   `output/` - This contains your jsonl (json lines) output files. It will be be empty in the remote bundle, but contain a `.gitkeep`. These are your input sources that have been converted into json lines.
```

Comment:
```suggestion
    -   `output/` - This contains your jsonl (json lines) output files. It will be be empty in the remote bundle, but contain a `.gitkeep`. These are your input sources that have been converted into json lines. Please note that you don't want to push student assessment output files to remote.
```